day_4/exercises/functions.js

```diff
@@ -14,17 +14,29 @@ printName();
 // Update the function below so that it takes an argument of your name and
 // prints your name
 
-function printName() { // hint.. add a parameter on this line :)
-  // YOUR CODE HERE
+function printName(myName) { // hint.. add a parameter on this line :)
+  console.log(myName);
 };
 
 printName("Albus Dumbledore");
+var myName = "Ashish";
+printName(myName);
 
 // Write a function that takes in 2 numbers as arguments and prints their sum, and
 // then call that function.
-//YOUR CODE HERE
+function findSum(num1, num2) {
+  console.log(num1 + num2);
+}
+var a = 1;
```

Comment:
Naming is an important part of being a strong developer. From a developer empathy perspective, I don't understand what `a` or `b` means just by reading the name of your variables. A better naming convention might be `number1` or `num2` or `firstVal` - being more semantic allows other developers to have a better understanding of the data you are working with!